conf/defaults.yaml

```diff
@@ -182,7 +195,7 @@ worker.log.level.reset.poll.secs: 30
 # control how many worker receiver threads we need per worker
 topology.worker.receiver.thread.count: 1
 
-task.heartbeat.frequency.secs: 3
+task.heartbeat.frequency.secs: 60
```

Comment:
This is a bit scary.  Why is the default now 20 times bigger than it was before?  If this is just for scale we should drop it back to the default and write up documentation on how to scale a cluster much larger.  Even in that case I don't feel comfortable with waiting 10+ mins for a worker to time out in the worst case.

Reply:
If that is the case can we at a minimum update the description of the config, or better yet create a new config that is specific to metrics and deprecate this one?  My biggest concern would be for running a 1.x topology on a 2.x cluster.  The 1.x topo may get some very odd behavior.

Reply:
Okey, i will  put a metric reporting interval conf item in.